frontend/packages/console-shared/src/components/dashboard/events-card/EventItem.tsx

```diff
@@ -1,13 +1,13 @@
 import * as React from 'react';
 import { RedExclamationCircleIcon } from '@console/shared';
-import { categoryFilter, getLastTime } from '@console/internal/components/events';
+import { typeFilter, getLastTime } from '@console/internal/components/events';
```

Comment:
note that this file will be removed (events card is replaced by activity card) so I did only minimal change to pass the compilation